app-emacs/ht/ht-2.3.ebuild

```
@@ -0,0 +1,22 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit elisp
+
+DESCRIPTION="The missing hash table library for Emacs."
```

[Avoid](https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Descriptive_text) full stops at the end of DESCRIPTION:
```suggestion
DESCRIPTION="The missing hash table library for Emacs"
```

(Not sure if this is in the devmanual, but if it isn't, it probably should be. Let me know!)